src/reanimated2/hook/useScrollViewOffset.ts

```diff
@@ -6,8 +6,8 @@ import type Animated from '../../index'; // TODO: fixme?
 import type { ScrollEvent } from './useAnimatedScrollHandler';
 import type { SharedValue } from '../commonTypes';
 import { findNodeHandle } from 'react-native';
-import { useEvent } from './utils';
 import { useSharedValue } from './useSharedValue';
+import { useEvent } from './useEvent';
```

Comment:
Please undo this to get smaller diff. It may be "nothing" but it simplifies usage of some git tools